[docs] update custom nginx configuration docs (#4583)
drud/ddev: docs/content/users/extend/customization-extendibility.md

## Custom nginx Configuration

> When you run [`ddev restart`](../usage/commands.md#restart) using `nginx-fpm`, DDEV creates a configuration customized to your project type in `.ddev/nginx_full/nginx-site.conf`. You can edit and override the configuration by removing the `#ddev-generated` line and doing whatever you need with it. After each change, run `ddev restart`. (For updates without restart see [Troubleshooting nginx Configuration](#troubleshooting-nginx-configuration))

Reviewer: Two little bits of punctuation:

```suggestion
When you run [`ddev restart`](../usage/commands.md#restart) using `nginx-fpm`, DDEV creates a configuration customized to your project type in `.ddev/nginx_full/nginx-site.conf`. You can edit and override the configuration by removing the `#ddev-generated` line and doing whatever you need with it. After each change, run `ddev restart`. (For updates without restart, see [Troubleshooting nginx Configuration](#troubleshooting-nginx-configuration).)
```

You can also have more than one config file in the `.ddev/nginx_full` directory, and each will be loaded when DDEV starts. This can be used for [serving multiple docroots](#multiple-docroots-in-nginx-advanced) and other techniques.

### Troubleshooting nginx Configuration

* Any errors in your configuration may cause the `web` container to fail and try to restart. If you see that behavior, use [`ddev logs`](../usage/commands.md#logs) to diagnose.

> * The configuration is copied into the container during restart. Therefore it is not possible to simply edit the host file for the changes to take effect. You may want to edit the file directly inside the container at `/etc/nginx/sites-enabled/`. (For example use [`ddev ssh`](../usage/commands.md#ssh) to get into the container)

Reviewer: “Run” the command, add punctuation:

```suggestion
* The configuration is copied into the container during restart. Therefore it is not possible to simply edit the host file for the changes to take effect. You may want to edit the file directly inside the container at `/etc/nginx/sites-enabled/`. (For example, run [`ddev ssh`](../usage/commands.md#ssh) to get into the container.)
```

* You can run `ddev exec nginx -t` to test whether your configuration inside the container is valid. (Or run [`ddev ssh`](../usage/commands.md#ssh) and run `nginx -t`.)

> * You can reload the nginx configuration by running either [`ddev restart`](../usage/commands.md#restart) or edit the configuration inside the container and run `ddev exec nginx -s reload` on the host system (inside the container just `nginx -s reload`).

Reviewer 2: If mentioning editing the config inside container, should give the path to it, thanks!

Author: Makes sense - fixed it!

> * You can reload the nginx configuration by running either [`ddev restart`](../usage/commands.md#restart) or edit the configuration inside the container at `/etc/nginx/sites-enabled/` and run `ddev exec nginx -s reload` on the host system (inside the container just `nginx -s reload`).

Reviewer: Keep verbs consistent:

```suggestion
* You can reload the nginx configuration by running either [`ddev restart`](../usage/commands.md#restart) or editing the configuration inside the container at `/etc/nginx/sites-enabled/` and running `ddev exec nginx -s reload` on the host system (inside the container just `nginx -s reload`).
```
